update.sh

```diff
@@ -1,20 +1,20 @@
-#!/bin/bash
+#!/usr/bin/env bash
 # ensure we have the most recent submodules
 git submodule foreach git pull origin master
 # update lts/nightly descriptions.
 
 # update them all in parallel...
-N=$(getconf _NPROCESSORS_ONLN)
+N=3
 for lts in {lts-haskell,stackage-nightly}/*.yaml
 do
-    $(find $NIX_TOOLS -type f -name "lts-to-nix") $lts > $(basename ${lts%.yaml}.nix) &
+    $(find -L $NIX_TOOLS -type f -name "lts-to-nix") $lts > $(basename ${lts%.yaml}.nix) &
     while [[ $(jobs -r -p | wc -l) -gt $N ]]; do
 	# can't use `wait -n` on older bash versions.
 	# e.g. what ships with macOS High Sierra
-	sleep 1;
+	wait -n;
```

Comment:
Now we've broken it on mac. I guess if we rely on some bash in nix, we have more control over the bash version?